src/field_specials.c

```diff
@@ -3216,52 +3254,53 @@ void sub_813ADD4(void)
     }
 }
 
-void sub_813AEB4(void)
+void GetBattleFrontierTutorMoveIndex(void)
 {
     u8 i;
-    u16 temp1 = 0;
-    u16 temp2 = 0;
+    u16 moveTutor = 0;
+    u16 moveIndex = 0;
     gSpecialVar_0x8005 = 0;
 
-    temp1 = VarGet(VAR_TEMP_E);
-    temp2 = VarGet(VAR_TEMP_D);
+    moveTutor = VarGet(VAR_TEMP_E);
+    moveIndex = VarGet(VAR_TEMP_D);
 
-    if (temp1 != 0)
+    if (moveTutor != 0)
     {
         i = 0;
         do
         {
-            if (gTutorMoves[i] == gUnknown_085B320C[temp2])
+            if (gTutorMoves[i] == sBattleFrontier_TutorMoves2[moveIndex])
```

Comment:
`gTutorMoves` should probably use the `TUTOR_MOVE_COUNT` constant in its definition.